docs/config.json

```diff
@@ -641,6 +641,10 @@
                 "enterprise"
               ]
             },
+            {
+              "title": "Joining Services via GCP",
```

Comment:
To standardize this section, I suggest we change this to:

```suggestion
              "title": "Via GCP",
```

And also update the line above, which is broken into three lines in the nav:

![image](https://github.com/gravitational/teleport/assets/2349184/12708119-e7c4-45b3-aae9-7b82296a26d0)